src/screens/Voucher/VoucherDetailScreen.tsx

```diff
@@ -82,25 +99,34 @@ export const VoucherDetailScreen = ({ route }: StackScreenProps<any>) => {
       <ImageSection mediaContents={mediaContents} />
 
       <Wrapper>
-        {!!quota &&
-          (maxQuantity && frequency ? (
-            <>
-              <BoldText small primary>
+        {!!quota && (
+          <>
+            {maxQuantity && frequency ? (
+              <>
+                <BoldText small primary={availableQuantity} placeholder={!availableQuantity}>
+                  {texts.voucher.detailScreen.limit(availableQuantity, maxQuantity)}
+                </BoldText>
+                <RegularText small primary={availableQuantity} placeholder={!availableQuantity}>
+                  {texts.voucher.detailScreen.frequency(maxPerPerson, frequency)}
+                </RegularText>
+              </>
+            ) : maxQuantity ? (
+              <BoldText small primary={availableQuantity} placeholder={!availableQuantity}>
                 {texts.voucher.detailScreen.limit(availableQuantity, maxQuantity)}
               </BoldText>
-              <RegularText small primary>
+            ) : (
+              <BoldText small primary={availableQuantity} placeholder={!availableQuantity}>
                 {texts.voucher.detailScreen.frequency(maxPerPerson, frequency)}
+              </BoldText>
+            )}
+
+            {!!dateOfAvailabilityText() && (
+              <RegularText small primary={availableQuantity} placeholder={!availableQuantity}>
+                {dateOfAvailabilityText()}
```

Comment:
please do not call the method twice to process it twice. call it once and use the returned value for the condition and the rendering.